docs/en/framework/api-development/dynamic-csharp-clients.md

```diff
@@ -163,6 +163,29 @@ context.Services.AddHttpClientProxies(
 
 `remoteServiceConfigurationName` parameter matches the service endpoint configured via `AbpRemoteServiceOptions`. If the `BookStore` endpoint is not defined then it fallbacks to the `Default` endpoint.
 
+#### Remote Service Configuration Provider
+You may need to get the remote service configuration for a specific remote service in some cases. For this, you can use the `RemoteServiceConfigurationProvider` interface.
+
+**Example: Get the remote service configuration for the "BookStore" remote service**
+
+````csharp
+public class MyService : ITransientDependency
+{
+    private readonly RemoteServiceConfigurationProvider _remoteServiceConfigurationProvider;
+
+    public MyService(RemoteServiceConfigurationProvider remoteServiceConfigurationProvider)
+    {
+        _remoteServiceConfigurationProvider = remoteServiceConfigurationProvider;
+    }
+
+    public async Task GetRemoteServiceConfiguration()
+    {
+        var configuration = await _remoteServiceConfigurationProvider.GetConfigurationAsync("BookStore");
```

Comment:
There is no `GetConfigurationAsync` method of `IRemoteServiceConfigurationProvider/RemoteServiceConfigurationProvider`